PocketKit/Sources/PocketKit/Settings/SettingsView.swift

```diff
@@ -20,12 +20,28 @@ struct SettingsView: View {
 
     var body: some View {
         VStack(spacing: 0) {
-            Form {
-//                Section(header: Text("Your Account").style(.settings.header)) {
-//                    PremiumRow(status: .notSubscribed, destination: EmptyView())
-//                    SettingsRowLink(title: "Reset Password", destination: EmptyView())
-//                    SettingsRowLink(title: "Delete Account", destination: EmptyView())
-//                }.textCase(nil)
+            if #available(iOS 16.0, *) {
+                SettingsForm(model: model)
+                    .scrollContentBackground(.hidden)
+            } else {
+                SettingsForm(model: model)
+            }
+        }
+        .navigationBarTitle("Settings", displayMode: .large)
+    }
+}
+
+struct SettingsForm: View {
+    @ObservedObject
+    var model: AccountViewModel
+    var body: some View {
+        Form {
+            Group {
+                //                Section(header: Text("Your Account").style(.settings.header)) {
```

Comment:
should we be keeping commented out code?

Reply:
this was pre-existing so I left it as-is, I think we want to comment it back in when we address the functionality of those settings
@kybhamilton correct me if I'm recalling wrong 🤔